package.json

```diff
@@ -23,20 +23,25 @@
   },
   "homepage": "https://github.com/vanwagonet/hexo-renderer-react",
   "dependencies": {
-    "babel-core": "^6.4.5",
-    "eval": "^0.1.0"
+    "@babel/core": "^7.17.10",
+    "@babel/preset-env": "^7.17.10",
+    "@babel/preset-react": "^7.16.7",
+    "@babel/register": "^7.17.7",
+    "eval": "^0.1.8",
+    "hexo": "^6.2.0",
+    "react": "^18.1.0",
+    "react-dom": "^18.1.0"
   },
   "peerDependencies": {
```

Comment:
I'm not sure if babel-related dependencies need to be included as peer dependencies. If do so, users should install babel manually, and the renderer will always use the user-installed version of babel, which usually the latest.